components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/docs/examples/alerts/alerts.yaml

```diff
@@ -0,0 +1,48 @@
+get:
+  - request:
+      method: GET
+      url: https://localhost:9443/api/am/publisher/v1.0/alert-types
+      headers: |
+          Authorization: Bearer eff01a97-4dc0-3542-a184-75aecf355ec2
+    response:
+      status:
+        code: 200
+        msg: OK
+      headers: |
+          Content-Type: application/json
+      body:
+        {
+          "count":2,
+          "alerts":[
+          {
+            "id":3,
+            "name":"AbnormalRequestTime",
+            "isRequireConfiguration":true
+          },
+          {
+            "id":4,
+            "name":"AbnormalResponseTime",
+            "isRequireConfiguration":true
+          }]
+        }
+  - rawRequest: |
+        GET https://localhost:9443/api/am/publisher/v1.0/alert-types HTTP/1.1
+        Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8
+    curl: |
+        curl -k -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8" https://localhost:9443/api/am/publisher/v1.0/alert-types
+    rawResponse: |
+      {
+        "count":2,
+        "alerts":[
+          {
+            "id":3,
+            "name":"AbnormalRequestTime",
+            "isRequireConfiguration":true
```

Comment:
shall we make it `requireConfiguration`?

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/7117

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/7117